auth-web/tests/unit/components/EntityManagement.spec.ts

```diff
@@ -114,4 +114,19 @@ describe('Entity Management Component', () => {
     expect(mockedNrMethod).toHaveBeenCalledTimes(0)
     expect(mockedPasscodeResetMethod).toHaveBeenCalled()
   })
+
+  it('all buttons, tooltips and v-menu selections exist', async () => {
+    // All buttons exist
+    expect(wrapper.find('#add-existing-btn').exists()).toBe(true)
+    expect(wrapper.find('#add-name-request-btn').exists()).toBe(true)
+    expect(wrapper.find('#incorporate-numbered-btn').exists()).toBe(true)
+
+    // Existing Business or NameRequest menu selections
+    wrapper.find('#add-existing-btn').trigger('click')
+    await Vue.nextTick()
+    expect(wrapper.findAll('.add-existing-item').length).toBe(2)
```

Comment:
findAll is depreciating but only when used on components it is still good to use on classes